modules/integration/tests/src/test/java/org/wso2/appmanager/integration/test/cases/publisher/webapp/lifecycle/management/ChangeStateFromApproveToPublishTestCase.java

```diff
@@ -39,6 +39,10 @@
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.assertTrue;
 
+/**
+ * This Test class verifies the ability of appCreator, appPublisher and admin users of changing web app life cycle
```

Comment:
Can we change the starting as "Test case which ..", because "This Test class ..." seems not beautiful.